docs/baking.md

```diff
@@ -220,25 +221,94 @@ In case you want to use a different alias for the baking account:
 2. update the `BAKER_ADDRESS_ALIAS` by editing the
     `/etc/default/tezos-baking-<network>` file.
 
-## Running baking instance on edo2net
+## Quick Start
```

Comment:
All LGTM now, however @itkach I recall you mentioning that this section could have been towards the top instead.

Alternatively it could be left where it is now, but instead add a link to "skip through" to it in the opening paragraph?